ENIGMAsystem/SHELL/Universal_System/instance_system.cpp

```diff
@@ -216,9 +216,19 @@ namespace enigma
   inst_iter *instance_event_iterator = &dummy_event_iterator; // Not bad for efficiency, either.
   object_basic *instance_other = NULL;
 
-  temp_event_scope::temp_event_scope(object_basic* ninst): oinst(instance_event_iterator->inst), oiter(instance_event_iterator), prev_other(instance_other)
-    { instance_event_iterator = &dummy_event_iterator; instance_event_iterator->inst = ninst; instance_other=ninst; }
-  temp_event_scope::~temp_event_scope() { instance_event_iterator = oiter; instance_event_iterator->inst = oinst; instance_other=prev_other; }
+  temp_event_scope::temp_event_scope(object_basic* ninst) :
+    oiter(instance_event_iterator),
+    prev_other(instance_other),
+    iter(ninst, NULL, NULL)
```

Comment:
For future reference, we're trying to move to Google style, which places opening curlies on the same line as the statement or function prototype.